Sources/SnowManLogic/ViewLogic/MockView.swift

```diff
@@ -14,6 +19,19 @@ public class MockView: Viewable {
 		displayLosingMessageCalled = false
 	}
 
+	public func promptNumberOfGuesses() {
+		self.promptNumberOfGuessesCalled = true
```

Comment:
This is called in the main. I saw your note further down about it being weird in the main. Let me consider that to solve this.